es/opendistro/api.py

```diff
@@ -20,8 +25,8 @@ def connect(
     scheme: str = "https",
     user: Optional[str] = None,
     password: Optional[str] = None,
-    context: Optional[Dict] = None,
-    **kwargs: Any,
+    context: Optional[Dict[Any, Any]] = None,
+    **kwargs: Dict[str, Any],
```

Comment:
nit: I think the original formulation was correct.
```suggestion
    **kwargs: Any,
```